### DISCUSSION TITLE
Share base [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6212] [Discord Threads]
> 🙋‍♂️ **letourpowerscombine**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Shared base is public and can be either with editor permissions or viewer permissions.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1129885278128050247)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
